app/yarn.lock

```diff
@@ -0,0 +1,7975 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Either we use yarn or npm. Having 2 lock files is crazy. I vote for npm as yarn was a thing in npm's early days. Nowadays npm can do everything and is the default tool in this ecosystem.

Reply:
yarn related files and code were removed.